invenio_app_ils/records/jsonresolver/document.py

```diff
@@ -29,23 +29,34 @@ def circulation_resolver(document_pid):
         item_search = ItemSearch()
         loan_search = IlsLoansSearch()
 
-        all_items_count = item_search.search_by_document_pid(document_pid).execute().hits.total
-        unavailable_items_count = item_search.get_unavailable_items_by_document_pid(document_pid).execute().hits.total
+        all_items_count = item_search.search_by_document_pid(
```

Comment:
This is why I really don't like 79 line length in 2019 when most of developers have something like [this](https://cdn.wccftech.com/wp-content/uploads/2018/03/Philips-492P8.jpg) (kidding).